### PR TITLE
Losses and gains

### LINKED ISSUE #295: Treatment of losses
Currently boundary losses are added to `e_track` or `p_track`. It would be nicer to add another field to the output file called losses. This field could also include losses in the interior due to inaccuracies (numerical or due to discontinuitities in input data).

### PR DESCRIPTION
**Pull request by ruudvdent**
bookkeeping of losses and gains. closes #295 #300

### CODE REVIEW

**Comment by ruudvdent** [comment]
closes # 295 #300

File: src/wam2layers/tracking/backtrack.py [hunk 1]
```diff
@@ -61,6 +61,15 @@ def backtrack(
         + region * precip * s_lower / (s_upper + s_lower)
         - evap * s_track_relative_lower
     )
+    # TODO: find better way to deal with negative values
```
Comment (Imme1992) [inline src/wam2layers/tracking/backtrack.py:64]: remove line 64 as we have addressed this now?

File: src/wam2layers/tracking/backtrack.py [hunk 2]
```diff
@@ -70,31 +79,47 @@ def backtrack(
         )
         + region * precip * s_upper / (s_upper + s_lower)
     )
+    # TODO: find better way to deal with negative values
```
Comment (Imme1992) [inline src/wam2layers/tracking/backtrack.py:82]: Remove line 82 as we have addressed this now

Comment (ruudvdent) [reply src/wam2layers/tracking/backtrack.py]: done

File: src/wam2layers/tracking/backtrack.py (hunk 1 above)
Comment (ruudvdent) [reply src/wam2layers/tracking/backtrack.py]: done

**Review by Peter9192 — APPROVED** [review]
Nice work. Can be polished a bit in the future.

**Pull request merged**